workflow/rules/common.smk

```diff
@@ -14,6 +14,16 @@ from hydra_genetics.utils.misc import extract_chr
 from hydra_genetics.utils.units import *
 from hydra_genetics.utils.samples import *
 
+
+def generate_read_group(wildcards):
```

Comment:
I would keep the functions in the bottom part of `common.smk` and have the whole config/setup in the beginning 😉

Reply:
I'm up for setting guidelines for this and bottom works for me.